.gitignore

```diff
@@ -2,4 +2,6 @@ VERSION
 *.log
 *.nupkg
 obj
+
 .history
+.vscode/settings.json
```

Comment:
Please undo this change.  This VSCode workspace settings file is specifically configured for this project.

Reply:
Sorry. I thought that I introduced the .vscode files. Done.